client/pd_service_discovery.go

```diff
@@ -124,10 +124,8 @@ type ServiceDiscovery interface {
 
 // ServiceClient is an interface that defines a set of operations for a raw PD gRPC client to specific PD server.
 type ServiceClient interface {
-	// GetAddress returns the address information of the PD server.
+	// GetAddress returns the address with HTTP scheme of the PD server.
```

Comment:
seems like will return https scheme?

Reply:
It should return addresses without the scheme.